config/initializers/sidekiq.rb

```diff
@@ -1,6 +1,13 @@
 if defined? Sidekiq
   Sidekiq.configure_server do |config|
     config.redis = { url: "redis://#{ENV['REDIS_HOST']}:#{ENV['REDIS_PORT']}/1" }
+
+    # we need to increase the db connection pool size to have enough connections
+    db_config = ActiveRecord::Base.configurations[Rails.env]
+    # the concurrency + 2 comes from: https://github.com/mperham/sidekiq/blob/master/4.0-Upgrade.md#whats-new
```

Comment:
Hmm, the +2 might be for redis.  But we do want at least one connection per sidekiq thread, right?